### PR TITLE
Updated POTFILES.in

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

File: data/guake-prefs.desktop.in [hunk 1]
```diff
@@ -1,10 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=Guake Preferences
-Name[pt_BR]=Preferências do Guake
-Comment=Configure your Guake sessions
-Comment[pt_BR]=Configurações do terminal Guake
-Comment[fr_FR]=Configurer votre session Guake
+_Name=Guake Preferences
+_Comment=Configure your Guake sessions
```
Comment (pypingou) [inline data/guake-prefs.desktop.in]: Shouldn't this be in english?

Comment (piotrdrag) [inline data/guake-prefs.desktop.in]: What do you mean? It is in English.

Comment (pypingou) [inline data/guake-prefs.desktop.in]: Sorry indeed, I think I wasn't quite awake and my eyes were not quite open, so basically  -ENOCOFFEE :)

Comment (piotrdrag) [inline data/guake-prefs.desktop.in]: No worries, happens to me too. :)

File: data/guake-autoconfigure.desktop.in [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
-Name=Autoconfigure Guake
-Comment=Autoconfigure the start of the Guake terminal
+_Name=Autoconfigure Guake
+_Comment=Autoconfigure the start of the Guake terminal
```
Comment (Stibbons) [inline data/guake-autoconfigure.desktop.in]: I don't understand why all these '_' ?

Comment (piotrdrag) [inline data/guake-autoconfigure.desktop.in]: _ makes gettext pick up these strings for translation. I'm not sure if there is any additional automake magic to make it work, but that's the basics.

Comment (Stibbons) [inline data/guake-autoconfigure.desktop.in]: ok

**Comment by Stibbons** [comment]
I revert this patch following issue #500.

**Pull request merged**